Sources/SourceKit/sourcekitd/SwiftCommand.swift

```diff
@@ -0,0 +1,122 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SKSupport
+import LanguageServerProtocol
+import Foundation
+
+/// The set of known Swift commands.
+///
+/// All commands from the Swift LSP should be listed here.
+public let builtinSwiftCommands: [String] = []
+
+/// A `Command` that should be executed by Swift's language server.
+public protocol SwiftCommand: Codable, Hashable {
```

Comment:
It looks like none of this is used yet. The basic design looks okay, but I'd like to leave this out until we have code that uses it (basically defer it to the next PR).  WDYT?